db/migrations/20240930100434_add_snap_uuid_to_templates.up.sql

```diff
@@ -1,6 +1,6 @@
 BEGIN;
 
-ALTER TABLE templates_repository_configurations ADD COLUMN IF NOT EXISTS snapshot_uuid UUID;
+ALTER TABLE templates_repository_configurations ADD COLUMN IF NOT EXISTS snapshot_uuid UUID NOT NULL;
```

Comment:
you'd have to set it to NOT NULL after the update below

in fact you may want to do the update to set the values, and then delete any null row to be safe and only then set it to NOT NULL.

Reply:
ohhh right, makes sense! just pushed up that fix